### PR TITLE
added upcoming React Session by Sagar

### PR DESCRIPTION
**Pull request by ankitt8**

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
 	<meta charset="UTF-8" />
 	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-	<meta property="og:title" content="Web Mini-Conference">
-	<meta property="og:description" content="Conference on important concepts a Web Developer should know about.">
-	<meta property="og:image" content="https://realdevsquad.com/img/events/og-image.png">
+	<meta property="og:title" content="Upcoming Events">
+	<meta property="og:description" content="Events on important concepts a Web Developer should know about.">
+	<!-- <meta property="og:image" content="https://realdevsquad.com/img/events/og-image.png"> --> <!--Don't have image for events-->
```
Comment (ankushdharkar) [inline events.html:10]: Use the poster image for the event please

File: events.html [hunk 2]
```diff
@@ -21,265 +21,61 @@
...
 		<a href="https://realdevsquad.com/" rel="noopener noreferrer">
 			<img class="logo-img" src="img/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
 		</a>
-	</header>
-
-	<main>
-		<section class="event-details">
-			<h2 class="event-upcoming">Upcoming Event</h2>
-			<h1 class="event-title">Web Mini-Conference</h1>
-			<h3 class="event-date-time">July 12th, 2pm IST</h3>
```
Comment (ankushdharkar) [inline events.html:37]: We need these for Sagar's session also please

File: index.html [hunk 3]
```diff
@@ -14,7 +14,10 @@
 
   <main>
     <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
-    <h2 class="info-note highlighted-text">You can check out all our current events happening <a href="./events.html">here</a></h2>
+    <h2 class="info-note highlighted-text">You can check out all our events below!!<br>
+      <a href="./events.html">Upcoming Events</a><span>  &nbsp;</span>
+       <a href="./web-mini-conf-july-2020.html">Web-Mini-Conf-July-2020</a>
+      </h2>
```
Comment (ankushdharkar) [inline index.html:20]: Cool

File: events.html (hunk 1 above)
Comment (ankitt8) [reply events.html]: Done

**Pull request merged**